CHANGELOG.md

```diff
@@ -9,11 +9,13 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 ### Added
 - `range` field for the forms.
 - `singleSubmit` attribute on all fields to allow only one submit per form to be used as calculation form.
-- "Result output" custom post type.
+- Result output custom post type.
```

Comment:
These two words put together without context are confusing, I'd prefer if they were accented in some way

Reply:
You can just _italic it_ 😄